source/MaterialXRenderGlsl/TextureBaker.h

```diff
@@ -203,13 +209,12 @@ class MX_RENDERGLSL_API TextureBaker : public GlslRenderer
     /// Optimize baked textures before writing.
     void optimizeBakedTextures(NodePtr shader);
 
-    /// Write the baked material with textures to a document.
-    DocumentPtr bakeMaterial(NodePtr shader, const StringVec& udimSet);
+    // TODO: Write images to memory rather than to disk
```

Comment:
Thanks for adding this TODO, which is an important reminder of the future work needed here.  To keep the public API of this method simple and straightforward to read, I would recommend moving the TODO into the method itself, just above the block of code that handles the writing of baked images to disk.